src/main.cpp

```diff
@@ -235,11 +241,31 @@ int main(int argc, char **argv) {
 		return EXIT_FAILURE;
 	}
 
+   PlayerInputComponent* playerInputComp = new PlayerInputComponent();

```

Comment:
From line 244-261 I didn't know if we should leave in main or drop into another function? I personally think it is ok here since the camera needs the player and the game manager needs the camera etc. What do you guys think?